tests/Doctrine/Tests/KeyValueStore/Storage/DynamoDbTest.php

```diff
@@ -310,11 +310,13 @@ public function testPassingArrayAsKeyIsAPassthruToInsert()
     public function testTryingToFindAnItemThatDoesNotExist()
     {
         $client = $this->getDynamoDbMock(['getItem']);
+        $result = $this->getDynamoDbResultMock(['get']);
```

Comment:
[$this->client->getItem()](https://github.com/doctrine/KeyValueStore/blob/master/lib/Doctrine/KeyValueStore/Storage/DynamoDbStorage.php#L277) does not return null on a missing item, but instead is returning an object of Aws\Result.

So I am not sure if it makes sense to keep the original test unless null is actually returned there in reality.

It makes me wonder why it was there in the first place, unless an older version of the SDK once returned null on missing items. With AWS SDK v3 at least, I am getting the Aws\Result object.